atlp-rwanda/e-commerce-lydia-32-fn
# ft/seller-statistics-timeframe#187570861

### What does this PR do?
This PR implements a new feature that allows sellers to view statistics on their sold products and earnings within a specified time frame.
### Description of Task to be completed?

- Created a new SellerStats component to display seller statistics, products sold and money made 
- Added an API endpoint (getSellerStats) to fetch seller statistics from the backend
- Integrated the new component with the existing seller dashboard

### How should this be manually tested?

1. Log in as a seller
2. Navigate to the seller dashboard
3. Look for a new "Sales Statistics" or "Earnings Report" section
4. Verify that the component displays:

- A list of sold products with quantities and prices
- Total earnings

### Any background context you want to provide?
This feature aims to provide sellers with clear, actionable data to help them make informed decisions about their product offerings and pricing strategies.
#### N/A
